webpack.common.js

```diff
@@ -31,59 +30,40 @@ if (WEBPACK_CUSTOM_ADD_PATH_CONFIG) {
   WEBPACK_CUSTOM_ADD_PATH = path.resolve(ROOT_DIR_CONFIG + '/' + WEBPACK_CUSTOM_ADD_PATH_CONFIG)
   WEBPACK_CUSTOM_ADD_PATH_DESC = WEBPACK_CUSTOM_ADD_PATH
 }
-const REPLACE = THEME_DIR + '/'
-console.log('--------------------------------')
-console.log('CONFIG (* = required) ')
-console.log('with current setting and example:')
-console.log('--------------------------------')
-console.log('FROM (*):      ' + THEME_DIR_CONFIG.replace(path.resolve(ROOT_DIR_CONFIG), './'))
-console.log('               --theme_dir=themes/mytheme # NB this dir should contain a src folder')
-console.log('               --theme_dir=vendor/vendor-name/package-name/client # NB this dir should contain a src folder')
-console.log('')
-console.log('--------------------------------')
-console.log('RESULTING (THEME OR VENDOR PACKAGE) CLIENT DIR')
+
 console.log('--------------------------------')
-console.log('' + REPLACE)
-console.log('')
+console.log('CONFIG (* = required)')
 console.log('--------------------------------')
-console.log('js_file:       ' + JS_FILE.replace(REPLACE, './'))
-console.log('               --js_file=' + JS_FILE_CONFIG)
-console.log('')
-console.log('css_file:      ' + CSS_FILE.replace(REPLACE, './'))
-console.log('               --css_file=' + CSS_FILE_CONFIG)
-console.log('')
+console.log('* FROM:      theme_dir: ' + THEME_DIR + ' e.g...')
+console.log('  --theme_dir=themes/mytheme (this dir should contain a src folder)')
+console.log('  --theme_dir=vendor/vendor-name/package-name/client (this dir should contain a src folder)')
+
+console.log('USING JS:    js_file:   ' + JS_FILE + ' set using --js_file=' + JS_FILE_CONFIG)
+console.log('USING CSS:   css_file:  ' + CSS_FILE + ' set using --css_file=' + CSS_FILE_CONFIG)
 console.log('--------------------------------')
```

Comment:
please dont revert this, I spent a lot of time making this better.

Reply:
fixed